### ISSUE TITLE
CssSyntaxError processing custom fonts (svg/eot/ttf/woff)

### ISSUE THREAD

**Issue by davidgarsan** [salsify/ember-css-modules#13]
Broccoli dependence fails as soon as a custom font is found:

```
CssSyntaxError: /PATH/tmp/cssmodules-input_base_path-nX6Fimql.tmp/0/poc-css-modules/styles/fonts/new-fontface.svg:1:1: Unknown word
    at Input.error (/PATH/node_modules/ember-css-modules/node_modules/broccoli-css-modules/node_modules/css-modules-loader-core/node_modules/postcss/lib/input.js:61:21)
...

The broccoli plugin was instantiated at: 
    at CSSModules.Plugin (/PATH/node_modules/ember-css-modules/node_modules/broccoli-css-modules/node_modules/broccoli-caching-writer/node_modules/broccoli-plugin/index.js:10:31)
    at CSSModules.CachingWriter [as constructor] (/PATH/node_modules/ember-css-modules/node_modules/broccoli-css-modules/node_modules/broccoli-caching-writer/index.js:21:10)
    at new CSSModules (/PATH/node_modules/ember-css-modules/node_modules/broccoli-css-modules/index.js:16:10)
    at ModulesPreprocessor.toTree (/PATH/node_modules/ember-css-modules/lib/modules-preprocessor.js:45:22)
    at /PATH/node_modules/ember-cli/node_modules/ember-cli-preprocess-registry/preprocessors.js:184:26
    at Array.forEach (native)
    at processPlugins (/PATH/node_modules/ember-cli/node_modules/ember-cli-preprocess-registry/preprocessors.js:182:11)
    at module.exports.preprocessJs (/PATH/node_modules/ember-cli/node_modules/ember-cli-preprocess-registry/preprocessors.js:175:10)
    at EmberApp.appAndDependencies (/PATH/node_modules/ember-cli/lib/broccoli/ember-app.js:1022:25)
    at EmberApp.javascript (/PATH/node_modules/ember-cli/lib/broccoli/ember-app.js:1137:34)

```

**Comment by dfreeman** [comment]
Thanks for reporting this! I've got a branch right now that reworks some of the pipeline internals that will take care of this – I'm hoping to land it in the next day or two as I can find time around Emberconf.

**Comment by davidgarsan** [comment]
:+1:

**Comment by dfreeman** [comment]
I just published 0.3.0, which should take care of this and also includes a test to make sure we don't regress again.

**Comment by davidgarsan** [comment]
Thank you!
Now works like a charm.
